placement/placement.go

```diff
@@ -56,10 +59,21 @@ func NewPlacement() Placement {
 }
 
 // NewPlacementFromProto creates a new placement from proto.
-func NewPlacementFromProto(p *placementpb.Placement) (Placement, error) {
+func NewPlacementFromProto(pb proto.Message) (Placement, error) {
```

Comment:
Why do we need to pass in a `proto.Message` instead of `placementpb.Placement`?

Reply:
Was trying to save statsdex from knowing the exact type of proto for constructing a Placement, but on the other hand since we already expose the protobuf type on the Placement interface, I guess the abstraction does not give too much value now.